contents/docs/advanced/proxy/vercel.mdx

```diff
@@ -10,8 +10,8 @@ Vercel supports [rewrites](https://vercel.com/docs/concepts/projects/project-con
 {
   "rewrites": [
     {
-      "source": "/ingest/:match*",
-      "destination": "https://app.posthog.com/:match*"
+      "source": "/ingest/:path*",
+      "destination": "https://app.posthog.com/:path*"
```

Comment:
We might want to recommend this to use the following source: `"source": "/ingest/:path(.*)"`, because it seems to fix the issue when using SvelteKit ([question](https://posthog.com/questions/post-hog-with-svelte-kit-and-vercel))

Reply:
Yeah, I'll add this.